test/test_helpers/integration-helpers.ts

```diff
@@ -47,11 +47,17 @@ export async function initFileStructure(files) {
   };
 }
 
-export async function createProject(files, connection: MessageConnection): Promise<{ normalizedPath: string; result: UnknownResult; destroy(): void }> {
+export async function createProject(
+  files,
+  connection: MessageConnection,
+  projectName?: string
+): Promise<{ normalizedPath: string; result: UnknownResult; destroy(): void }> {
   const dir = await createTempDir();
 
   dir.write(files);
-  const normalizedPath = path.normalize(dir.path());
+  const normalizedPath = projectName ? path.normalize(path.join(dir.path(), projectName)) : path.normalize(dir.path());
+
+  await initDynamicServer(connection, normalizedPath);
```

Comment:
basically, `els.registerProjectPath` should add project, if it not working we should check server.ts internals to get it working, we could initialize server only once

Reply:
could you help look into this one?